how-it-works/00_General/01_overview-of-the-internet-computer.subpage.md

```diff
@@ -0,0 +1,125 @@
+Using Internet Identity, canisters can authenticate users based on biometry using the WebAuthn protocol.
+
+## Core Internet Computer Protocol
+
+Each subnet of the IC is driven by the core Internet Computer protocol (core IC protocol) an implementation of which is running on every node.
+This protocol consists of the following four layers: (1) The peer-to-peer, (2) consensus, (3) message routing, and (4) execution layer.
+This core IC protocol stack is running on all nodes of any subnet and drives the subnet to make progress in terms of consensus and message execution.
+Each subnet of the IC thereby is its own replicated state machine that makes progress independently of the other subnets of the IC.
+
+## Chain-Key Cryptography
+
+Many parts of the protocol of the IC depend on *chain-key cryptography*, also referred to as chain-key technology.
+Chain-key cryptography is the collection of cryptographic mechanisms that enable the Internet Computer Protocol.
+
+One important mechanism in this toolbox is non-interactive distributed key generation (NIDKG) for BLS signature keys.
+NIDKG is one of the most involved technical components of the IC and allows for the secure and trusteless setup of the signing key of a new subnet.
+A private key secret-shared with NIDKG is re-shared periodically so that compromised key shares become worthless and adaptive attacks become harder.
+BLS threshold signatures are another important tool in this toolbox and are used, for example, to certify the state of a subnet and to create unpredictable and unbiasable random numbers.
+
+Another new mechanism in the toolbox are threshold ECDSA signatures and their interactive distributed key generation protocol.
+Threshold ECDSA allows canister smart contracts to request signatures, which are then computed jointly by a subnet, based on a secret-shared key that no single party knows.
+
+## Chain-Evolution Technology
+
+Chain-evolution technology refers to specific mechanims that enable the IC to operate in the long term.
+To this end, chain-evolution technology enables new nodes to join a subnet or nodes that have been down to catch up with the remaining part of the subnet.
+All this can be done efficiently, without a node needing to replay all messages from the start of its subnet.
+
+## Ingress Messages
+
+When a user sends a message to a canister from outside the IC, this message goes through the core IC protocol stack until it gets executed.
+Successful execution can change the state of the subnet and results in the generation of a response that is stored in a certified part of replicated state.
+The response can be retrieved by the user's client and verified for authenticity.
+This allows users to obtain certified answers to canister messages almost instantly.
+Getting an alomst-instant certified response to a request is a feature that sets apart the IC from other blockchain projects.
+
+## Governance
```

Comment:
As above, I think it makes most sense to focus on the NNS here, and move the SNS to a different page.